### PR TITLE
Start moving docs to website

### PR DESCRIPTION
**Pull request by mnonnenmacher**
See the commit messages for details.

### CODE REVIEW

File: docs/api/trigger_run_parameters.md [hunk 1]
```diff
@@ -19,34 +20,98 @@ So the job configurations are typically not a suitable model for end users to sp
...
-* Further parameters related to FossID, e.g. the project name or the number of delta scans to keep, are also added to the FossID options in the scanner configuration.
-* The _applicationCategory_ parameter is also checked for correctness. In this example, the category determines the Evaluator configuration to be used - the rule set and the license classifications. The validation script therefore creates an `EvaluatorJobConfiguration` and populates it accordingly.
+* Further parameters related to FossID, e.g., the number of delta scans to keep, are also added to the FossID options in the scanner configuration.
+* The _applicationCategory_ parameter is also checked for correctness. In this example, the category determines the Evaluator configuration to be used: the rule set and the license classifications. The validation script therefore creates an `EvaluatorJobConfiguration` and populates it accordingly.
```
Comment (sschuberth) [inline docs/api/trigger_run_parameters.md]: I find this sentence before and after hard to read. Maybe "to be used, i.e. the" would be better?

File: docs/api/trigger_run_parameters.md [hunk 2]
```diff
@@ -19,34 +20,98 @@ So the job configurations are typically not a suitable model for end users to sp
...
+
+* check whether certain parameters have a defined set of values; e.g., only rule sets "standard", "web", "special" are allowed
+* map the values given by the user to other values; e.g., the rule set "standard" could be mapped to a specific configuration file path from where it can be downloaded
+* create specific job configurations for parameters; e.g., the users can specify a set of FossID parameters, which are then added to the corresponding options in the scanner configuration
+
+For example, a validation script could use the parameters to configure the scanner and evaluator jobs.
+
+The following input:
```
Comment (sschuberth) [inline docs/api/trigger_run_parameters.md]: Is this now the replacement for the `config_validation_example.puml`? According to the commit message, I would have expected a Mermaid diagram.

File: website/docusaurus.config.ts [hunk 3]
```diff
@@ -31,6 +31,10 @@ const config: Config = {
     locales: ['en'],
   },
 
+  markdown: {
```
Comment (sschuberth) [inline website/docusaurus.config.ts:34]: Consider moving this commit to before Mermaid diagrams are actually introduced.

File: website/docs/guides/parameter-validation-script.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
 
-* When specifying the list of _advisors_ to run, their configuration is expected. This includes information like URLs of external systems and access credentials.
-* Some workers rely on plugins that implement the actual functionality. Those plugins expect their configuration in maps containing key-value pairs; and the whole plugin configuration is a map that uses the plugin names as keys and the plugin options as values. So, the name of plugins and their supported options must be known.
-* The rules to be processed by the Evaluator must be loaded from somewhere. Where are they located and how can they be accessed?
-* The situation for template files to be consumed by the Reporter is similar; they also need to be retrieved from a specific location.
+- When specifying the list of _advisors_ to run, their configuration is expected. This includes information like URLs of external systems and access credentials.
```
Comment (sschuberth) [inline website/docs/guides/parameter-validation-script.md:9]: Strictly speaking, changing the bullet list markers is unrelated to moving files around.

File: website/docs/guides/parameter-validation-script.md [hunk 5]
```diff
@@ -55,9 +55,7 @@ Could be transformed to these job configurations:
 {
   "resolvedJobConfigs": {
     "scanner": {
-      "scanners": [
-        "FossId"
-      ],
+      "scanners": ["FossId"],
```
Comment (sschuberth) [inline website/docs/guides/parameter-validation-script.md:58]: This is strictly speaking an unrelated formatting change as well.

File: docs/api/trigger_run_parameters.md (hunk 2 above)
Comment (mnonnenmacher) [reply docs/api/trigger_run_parameters.md]: Yes, I have clarified it in the commit message now. It makes more sense to me to show the job configurations in JSON like when using the API instead of using a diagram.

File: website/docusaurus.config.ts (hunk 3 above)
Comment (mnonnenmacher) [reply website/docusaurus.config.ts:34]: It is before introducing them to the website, the previous conversion is still happening in the docs folder.

File: website/docs/guides/parameter-validation-script.md (hunk 4 above)
Comment (mnonnenmacher) [reply website/docs/guides/parameter-validation-script.md:9]: The reformatting is required because the website uses prettier for the formatting like the ui does. I have added this to the commit message now.

**Review by sschuberth — APPROVED** [review]
